app/assets/admin/js/settings.js

```diff
@@ -979,6 +989,25 @@ jQuery( document ).ready( function ( $ ) {
 		window.location.href = '/wp-admin/tools.php?page=remove_personal_data';
 	});
 
+	/**
+	 * Show error message if validation fails.
+	 *
+	 * @param selector
+	 * @param error_msg
+	 * @returns {boolean}
+	 */
+	function rtp_show_error_message( selector, error_msg ) {
+		var elm_selector = jQuery( selector );
+		elm_selector.focus();
+		elm_selector.css( 'border-color', 'red' );
+		if ( elm_selector.parent().length > 0 && 'error_msg' !== elm_selector.parent().attr( 'class' ) ) {
```

Comment:
Take variable for `elm_selector.parent()`and use it to avoid duplicate DOM queries.